ios/ShoutemApp/AppDelegate.m

```diff
@@ -43,7 +43,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
                                                       moduleName:@"ShoutemApp"
                                                initialProperties: initialProperties
                                                    launchOptions:launchOptions];
-  rootView.backgroundColor = [UIColor clearColor];
+  rootView.backgroundColor = [UIColor whiteColor];
```

Comment:
it would be cool if we could write this color in some .plist while building the app, so it would have a colored background. So, this will actually prevent the app from having a black screen and it is going to have the white one while reloading?